publisher/proto/publisher.proto

```diff
@@ -10,7 +10,6 @@ message Request {
   string LogURL = 2;
   string LogPublicKey = 3;
   bool precert = 4;
-  bool storeSCT = 5;
```

Comment:
Please leave behind a comment indicating that field 5 is reserved; the next new field should be 6.